charge.go

```diff
@@ -344,6 +345,13 @@ type ChargePaymentMethodDetailsEps struct {
 	VerifiedName string `json:"verified_name"`
 }
 
+// ChargePaymentMethodDetailsFpx represents details about the FPX PaymentMethod.
+type ChargePaymentMethodDetailsFpx struct {
+	AccountHolderType string `json:"account_holder_type"`
```

Comment:
This needs to be an "enum" not a string on the resource. You would use the same enum that you'd have on the PaymentMethod one really. You'd do something like this: https://github.com/stripe/stripe-go/blob/master/paymentmethod.go#L5-L12

You'd add a `PaymentMethodFpxAccountHolderType` constant in `payment_metod.go` and use it on the resource (params are still `*string`)

Reply:
Done @remi-stripe

Reply:
AFAICT it's still not done here, did you not push your commit? It still says `string`